src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -269,11 +269,29 @@ export function EmbeddedPositionSearchPanel({
                             ...slotProps?.searchRoot?.sx
                         })
                 }}>
-                <CollapsibleGroup sx={{border:0, padding:0}} >
+                <CollapsibleGroup
+                    sx={{borderWidth: '0px',
+                        '& .MuiAccordionSummary-root': {
+                            paddingBlockStart: '0.5rem',
+                            paddingBlockEnd: '0.5rem',
+                            minBlockSize: '1rem',
+                            '&.Mui-expanded': {
+                                height: '1rem '
+                            }
+                    }}}>
```

Comment:
@loitly do you mind reviewing this latest commit? it controls the styling for the header when the search panel is open (the idea was to make it as small as possible, as the scientists don't want the search panel to take up too much space) but still keep it collapsible. 

Feel free to review other stuff too, but if this looks ok I can go ahead and merge this PR.